parser_2gis/writer/models/schedule.py

```diff
@@ -67,7 +67,7 @@ def to_str(self, join_char: str, add_comment: bool = False) -> str:
         Returns:
             Schedule as a string.
         """
-        days_names = [x.name for x in self.__fields__.values() if x.type_ == ScheduleDay]
+        days_names = [x.name for x in self.__fields__.values() if type(x) == ScheduleDay]
```

Comment:
I am not quite sure what version of Pydantic you use, but `type(x)` should always return `pydantic.fields.ModelField`, that's why there is special attribute `type_` for the fields.